src/test/java/com/nvidia/spark/rapids/jni/CastStringsTest.java

```diff
@@ -59,6 +59,60 @@ void castToIntegerTest() {
     }
   }
 
+
+  //This is for testing the CastStrings class with the input of "inf" etc string
+  @Test
+  void castToFloatsInfTest(){
+    // The test data: Table.TestBuilder object with a column containing the string "inf"
+    Table.TestBuilder tb2 = new Table.TestBuilder();
+    tb2.column("INFINITY ", "inf", "+inf ", " -INF  ");
+
+    Table.TestBuilder tb = new Table.TestBuilder();
+    tb.column(Float.POSITIVE_INFINITY, Float.POSITIVE_INFINITY, Float.POSITIVE_INFINITY, Float.NEGATIVE_INFINITY);
```

Comment:
Let's test double type too.

Reply:
Some negative tests would be nice. Some strings like `INFINITY AND BEYOND`.

Reply:
`INFINITY AND BEYOND`  added.
with aother one test string `INF`.